src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -634,15 +642,26 @@ ApiGlobe.prototype.setCenter = function setCenter(coordinates, isAnimated) {
  * The zoom level and the scale can't be set at the same time.
  * <iframe width="100%" height="400" src="//jsfiddle.net/iTownsIGN/7yk0mpn0/embedded/" allowfullscreen="allowfullscreen" frameborder="0"></iframe>
  * @constructor
- * @param {Position} pPosition - The detailed position in the scene.
- * @param      {boolean}  isAnimated  Indicates if animated
- * @return     {Promise}
+ * @param {Position} pPosition - The detailed position in the scene : longitude, latitude, tilt, heading, range, level, scale. The range has priority over the level and scale. The level has priority over scale. For a scale of 1/500 it is necessary to write 0,002.
+ * @param {boolean}  isAnimated  Indicates if animated
+ * @return {Promise}
  */
 ApiGlobe.prototype.setCenterAdvanced = function setCenterAdvanced(pPosition, isAnimated) {
     isAnimated = isAnimated || this.isAnimationEnabled();
     return this.setCenter(pPosition, isAnimated).then(() => {
-        const p = this.scene.currentControls().setOrbitalPosition(undefined, pPosition.heading, pPosition.tilt, isAnimated);
-        return p;
+        if (pPosition.range === undefined) {
+            if (pPosition.level) {
+                pPosition.range = this.getRangeFromZoomLevel(pPosition.level);
+            } else if (pPosition.scale) {
+                pPosition.range = this.getRangeFromScale(pPosition.scale);
```

Comment:
> I think it is more explicit to give the scale param as a fraction denominator than in decimal (ex : 500 instead of 0.002)

The scale of a map is the ratio,  we must be find a another name to this parameter